modules/nw-multus-ipam-object.adoc

```diff
@@ -237,6 +244,38 @@ Whereabouts can be used for both IPv4 and IPv6 addresses.
 }
 ----
 
+[id="dynamic-ip-address-assignment-whereabouts-overlapping-ip-ranges_{context}"]
+=== Dynamic IP address assignment that uses Whereabouts with overlapping IP address ranges
+
+The following example shows a dynamic IP address assignment that uses overlapping IP address ranges for multi-tenant networks:
+
+.NetworkAttachmentDefinition 1
+[source,json]
+----
+{
+  "ipam": {
+    "type": "whereabouts",
+    "range": "192.0.2.192/29",
+    "network_name": "example_net_common", <1>
+  }
+}
+----
+<1> Optional. If set, must match the `network_name` of NetworkAttachmentDefinition 2.
+
+.NetworkAttachmentDefinition 2
+[source,json]
+----
+{
+  "ipam": {
+    "type": "whereabouts",
+    "range": "192.0.2.192/24",
+    "network_name": "example_net_common", <1>
+    "enable_overlapping_ranges": true
```

Comment:
I'm starting to wonder if we should put the details about the `enable_overlapping_ranges` in the release notes, and instead... Maybe offer some best practices?

For example...

> When creating multiple Whereabouts `range` CIDRs, it's considered a best practice to not use ranges that overlap one another. For example, instead of creating configurations that use both `192.0.2.0/29` and `192.0.2.0/24` instead, use the larger range.